src/sbus/request/sbus_request_sender.c

```diff
@@ -101,10 +101,16 @@ void
 sbus_senders_delete(hash_table_t *table,
                     const char *name)
 {
-    DEBUG(SSSDBG_TRACE_INTERNAL, "Removing identity of sender [%s]\n",
-          name);
+    hash_key_t table_key;
+
+    table_key.type = HASH_KEY_STRING;
+    table_key.str = discard_const_p(char, name);
 
-    sss_ptr_hash_delete(table, name, true);
+    if (hash_has_key(table, &table_key)) {
```

Comment:
I think you can use `sss_ptr_hash_has_key` instead to avoid created the key yourself?

Reply:
Thanks for the hint. I somehow overlooked this helper.
Updated.